jenkins/catapult/runDockerTests.py

```diff
@@ -74,9 +75,11 @@ def main():
 	compose_filename += '.yaml'
 	compose_template_filepath = Path(__file__).parent / 'templates' / compose_filename
 	print(f'processing template from {compose_template_filepath}')
+	compiler_filepath = Path(args.compiler_configuration)
+	compiler_base_path = f'{compiler_filepath.parent.name}/{compiler_filepath.name}'
```

Comment:
compiler_filepath is the full path from the git root
compiler_base_path is the compiler filename and its parent path.  so it would be like ``amd64/gcc-latest.yaml``